app/frontend/ComponentWrapper.js

```diff
@@ -1,4 +1,5 @@
 /* @flow */
+import 'babel-polyfill';
```

Comment:
This needs to go before any code so ContentWrapper is not the best place to put it (since components.js runs before and has its own imports). The best place to put it would be in our webpack config (as our first entry point).

For next release maybe.